### DISCUSSION TITLE
Heat recovered by recuperators' 1&2 in sCO2 cycle

### DISCUSSION THREAD

**Discussion by zain-oday** [oemof/tespy#466] [Q&A] [answered]
Hi, everyone.
In the first place, I'd like to express my gratitude for such useful software.
My question is about sCO2 power plant. After running the code and getting the results, I noticed that the amount of heat energy that was recovered by recuperators 1&2 was greater than the input value, as shown in attached file So I'm wondering why I get those values.
[sCO2_model_report.pdf](https://github.com/oemof/tespy/files/13812109/sCO2_model_report.pdf)
Paragraphs 3.1.2 and 3.4.2

**Comment by fwitte** [comment]
Hi @zain-oday,

thanks for reaching out, what is the code you are using for this? From the sCO2 exergy analysis repository?

Best

**Reply by zain-oday** [reply]
Thank you for response Mr. fwitte ,
Yes, it is. I will attach the zip folder with this comment.
best regards 
[sCO2_exergy-main.zip](https://github.com/oemof/tespy/files/13813725/sCO2_exergy-main.zip)

**Reply by fwitte** [reply]
Hi @zain-oday,

so the heat recuperated can be any value, it is not limited by what you put in if I understand this correctly. For example, you can put in 100 MW of thermal energy, recuperate/recycle 1000 MW internally and only extract 40 MW in a turbine, right? The total balance of the cycle still sums to zero.

**Reply by plevisse-naarea** [reply, marked as answer]
@fwitte 
Yes that's the correct way to interpret it : recuperators in sCO2 cycles tend to have heat transfers greater than the external heat input. Since it's an internal heat transfer, it cancels itself out in the heat balances.

**Reply by fwitte** [reply]
So, resolved? :)

**Reply by plevisse-naarea** [reply]
Yeap

**Comment by zain-oday** [comment]
Hi,
Dear Mr. @fwitte 
I hope you have a good day
Following up on my previous question, I noticed that the Sankey diagram shows that the heat rejected from the water cooler is returned back to the recuperators again while it is supposed to be reject to the environment, which explains why the energy recovered from the recuperators increases.
I look forward to response
best regards

**Reply by fwitte** [reply]
The sankey diagram shows exergy flows not energy (or heat), so it is a little bit difficult to compare here. But the reason for the recycling is, that after the CO2 leaves the water cooler, there is still exergy in the CO2 flow left (both, temperature and pressure are above ambient state). Would you have time to join one of the next tespy user meetings (January 15, 17:00 CET would be the next one), then we could discuss this topic in more detail.

Best
